turbinia/notify.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Sends email notifications'''
+
+import logging
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from turbinia import config
+log = logging.getLogger('turbinia')
+
+#Sends notfications via email
+def sendmail(subject, message):
+  '''Sends an Email notification via SMTP'''
+  try:
+    if config.EMAIL_NOTIFICATIONS is True:
+
+      #Puts message in MIME format
+      msg = MIMEMultipart()
+      msg['From'] = config.EMAIL_ADDRESS
+      msg['To'] = config.EMAIL_RECIEVING_ADDRESS
```

Comment:
(sorry if this is a dupe comment, I thought I added this before, but I don't see it now)

It would be nice to add this to the input parameters to the function so that you can pass in the `requester` info, and send the notification directly to the person who initiated the Turbinia request.   The `requester` will only be the username though, so you might need to add `EMAIL_DOMAIN` to the config.